tests/abi_parser.test.cpp

```diff
@@ -1,41 +1,43 @@
-#include "../include/abiparse.hpp"
```

Comment:
Nit: Change include order https://libhal.github.io/4.2/contributor_guide/style/#s14-headers

Also, don't use local path, include is already considered an include path 
ie #include "abiparse.hpp"